gulpfile.js

```diff
@@ -33,91 +37,104 @@ gulp.task('test', function(done) {
   runSequence('clean', 'test:node', 'test:browser', done);
 });
 
-
 gulp.task('hooks:precommit', ['build'], function() {
-  return gulp.src(['dist/*', 'lib/*'])
-    .pipe(plugins.git.add());
+  return gulp.src(['dist/*', 'lib/*']).pipe(plugins.git.add());
 });
 
 gulp.task('build:node', ['lint:src'], function() {
-    return gulp.src('src/**/*.js')
-        .pipe(plugins.babel())
-        .pipe(gulp.dest('lib'));
+  return gulp
+    .src('src/**/*.js')
+    .pipe(plumber())
+    .pipe(plugins.babel())
+    .pipe(gulp.dest('lib'));
 });
 
 gulp.task('build:browser', ['lint:src'], function() {
-  return gulp.src('src/browser.js')
-    .pipe(plugins.webpack({
-      output: { library: 'StellarBase' },
-      module: {
-        loaders: [
-          { test: /\.js$/, exclude: /node_modules/, loader: 'babel-loader'}
+  return gulp
+    .src('src/browser.js')
+    .pipe(plumber())
```

Comment:
I'm okay with leaving it out. I think we should add a separate command that you run during dev, since right now any error will crash the watcher. But we can tackle that separately.